meta-webos/recipes-multimedia/ffmpeg/ffmpeg_2.3.6.bb

```diff
@@ -42,7 +42,7 @@ FFMPEG_CONF = ' \
     --disable-pthreads \
     --disable-w32threads \
     --disable-os2threads \
-    --enable-thumb \
+     ${@bb.utils.contains('TUNE_FEATURES', 'thumb', ' --enable-thumb', '', d)} \
```

Comment:
Shouldn't it respect ARM_INSTRUCTION_SET as well?

Newer ffmpeg 3.3 (from Yocto 2.3) where thumb is explicitly disabled:

# Build fails when thumb is enabled: https://bugzilla.yoctoproject.org/show_bug.cgi?id=7717 ARM_INSTRUCTION_SET = "arm"

it's disabled because of thumb1 and thumb2 should work fine, but maybe we can drop this --enable-thumb completely.